### ISSUE TITLE
Stuck in infinite loading state when doing a POST

### ISSUE THREAD

**Issue by jsheffers** [tom-murray/Samsung-Art-Mode#1]
Using postman after starting a docker container on 5001 and hitting with my access_key and keywords.  It just spins and spins.  Any ideas for troublshooting?

**Comment by jsheffers** [comment]
![CleanShot 2024-12-03 at 00 10 14@2x](https://github.com/user-attachments/assets/fb620282-2f83-4a7e-8d4f-76debdcf0ae1)

**Comment by jsheffers** [comment]
![CleanShot 2024-12-03 at 00 11 08@2x](https://github.com/user-attachments/assets/6d57a2c0-d3d6-46f5-8220-b0aecd76a35a)

**Comment by tom-murray** [comment]
Apologies for the late reply, I'm assuming the container is on the same subnet as the tv? Do you get a prompt on the TV to allow the connection and is it showing under device list? You can find this under 'Settings' > 'General' > 'External Device Manager'.

**Comment by jsheffers** [comment]
Yes, the container is on the same subnet. I'll take a look at this later tonight to see if it's showing under the device manager. I don't think I got a prompt. I'll let you know. Thanks for the response.
